### PR TITLE
admin: Explain the official log formats

### PR DESCRIPTION
**Pull request by jfsmig**

### CODE REVIEW

File: doc/admin-guide/source/operations_log_format.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+   * - Return Code
+     - The numeric return code of the message.
+   * - Response Time
+     - How many microseconds it took to handle the request until a reply was ready (but not sent yet!)
+   * - Response Size
+     - How many bytes have been serialized for the reply. In case of HTTP requests, this doesn't include the headers.
+   * - User ID
+     - The ID of the end-user the request is issued for
+   * - Session ID
+     - The ID of the user's session, sometimes also named Request-I, used for aggregation purposes.
```
Comment (fvennetier) [inline doc/admin-guide/source/operations_log_format.rst]: "Request-I" -> "Request-Id"

File: doc/admin-guide/source/operations_log_format.rst [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+Log Format
+==========
+
+COMMON enveloppe
+~~~~~~~~~~~~~~~~
+
+The services in an OpenIO system respect a common format for their logs. Each
+log item is a single line of text, encoded in UTF-8. A line is a sequence of
+items separated by a sequence of white spaces. For an easy parsing, the variety
+of the messages is organized around a recursion of enveloppes (header and
```
Comment (fvennetier) [inline doc/admin-guide/source/operations_log_format.rst]: "enveloppes" -> "envelopes"
(appears several times)

**Comment by GuillaumeDelaporte** [comment]
You should put the sentence regarding our log format before the common envelope section.

An exemple in each section could be valuable as well.

Otherwise 👍

File: doc/admin-guide/source/operations_log_format.rst [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+   * - Level
+     - A value in the set: **ERR**, **WRN**, **NOT**, **INF**, **DBG**, **TR0**, **TR1**
+   * - Payload
+     - An arbitrary message.
+
+
+Example
+~~~~~~~
+
+Let's consider this following lines, and see how it decomposes.
```
Comment (vdombrovski) [inline doc/admin-guide/source/operations_log_format.rst]: Below is an example of a single line of logs, as well as its mapping to the composing parameters.

File: doc/admin-guide/source/operations_log_format.rst [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+.. code-block:: text
+
+    2017-04-25T17:00:01.094517+02:00 localhost OIO,OPENIO,meta0,1[12159]: 12159 1E9A access INF 127.0.0.1:6004 127.0.0.1:48780 M0_GET 200 89 91 - 742FBB9DC7674C7C7959957801F06B44 t=63 AAA0
+
+The first 3 fields are set by syslog, and that why the process ID is redundant
```
Comment (vdombrovski) [inline doc/admin-guide/source/operations_log_format.rst]: ~~and that why the process ID is redundant on that host~~ => making the field process ID redundant.
Alternative: remove the sentence, as it brings no relevant information.

File: doc/admin-guide/source/operations_log_format.rst [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+   * - Response Size
+     - `91`
+   * - User ID
+     - `-`
+   * - Session ID
+     - `742FBB9DC7674C7C7959957801F06B44`
+   * - Payload
+     - `t=63 AAA0`
+
+In this example, all the fields are always present as expected, but one
```
Comment (vdombrovski) [inline doc/admin-guide/source/operations_log_format.rst]: but one **of** the

File: doc/admin-guide/source/operations_log_format.rst [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+missing fields has been valuated by a dash. The final field is has an arbitrary
+(or unspecified) format, it depends from the service implementation.
+
+The key ``t=`` is used to explain the time spent by a worker thread, once the
```
Comment (vdombrovski) [inline doc/admin-guide/source/operations_log_format.rst]: ~~is used to explain~~ => represents the time (in [UNIT]) spent...

File: doc/admin-guide/source/operations_log_format.rst [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+request has been polled out of the queue in front of the thread pool.
+
+Another well-known key OpenIO SDS uses is ``e=`` to explain the root cause of
```
Comment (vdombrovski) [inline doc/admin-guide/source/operations_log_format.rst]: Another key used by OpenIO SDS is ``e=``, that gives the root cause...

File: doc/admin-guide/source/operations_log_format.rst [hunk 8]
```diff
@@ -0,0 +1,157 @@
+==========
+Log Format
+==========
+
+The services in an OpenIO system respect a common format for their logs. Each
+log item is a single line of text, encoded in UTF-8. A line is a sequence of
+items separated by a sequence of white spaces. For an easy parsing, the variety
+of the messages is organized around a recursion of envelopes (header and
+payload). All the fields are always present on a line, and when a field is not
+valuated it is represented by a single dash character.
```
Comment (vdombrovski) [inline doc/admin-guide/source/operations_log_format.rst]: ~~valuated~~ => set

File: doc/admin-guide/source/operations_log_format.rst [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+     - `91`
+   * - User ID
+     - `-`
+   * - Session ID
+     - `742FBB9DC7674C7C7959957801F06B44`
+   * - Payload
+     - `t=63 AAA0`
+
+In this example, all the fields are always present as expected, but one
+missing fields has been valuated by a dash. The final field is has an arbitrary
```
Comment (vdombrovski) [inline doc/admin-guide/source/operations_log_format.rst]: ~~valuated~~ =>  is defaulting to a dash

File: doc/admin-guide/source/operations_log_format.rst [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+Let's mention the case of the ACCESS log for outgoing requests. It is triggered
+by an option in the central configuration file, and the format of each line is
+exactly the same as for incmong requests, excepted that the Domain in the
+common envelope is set to ``out``.
+
+
+DEBUG logs
+~~~~~~~~~~
+
+Any service might also emit traces, are they impulsed by a request or by a
```
Comment (vdombrovski) [inline doc/admin-guide/source/operations_log_format.rst]: ~~are they impulsed~~ => that are generated

File: doc/admin-guide/source/operations_log_format.rst [hunk 11]
```diff
@@ -0,0 +1,157 @@
...
+     - The ID of the end-user the request is issued for
+   * - Session ID
+     - The ID of the user's session, sometimes also named Request-Id, used for aggregation purposes.
+   * - Payload
+     - An arbitrary payload, often organized as a sequence of ``key=value`` pairs.
+
+
+Let's mention the case of the ACCESS log for outgoing requests. It is triggered
+by an option in the central configuration file, and the format of each line is
+exactly the same as for incmong requests, excepted that the Domain in the
```
Comment (vdombrovski) [inline doc/admin-guide/source/operations_log_format.rst]: ~~incmong~~ => incoming?

Comment (vdombrovski) [inline doc/admin-guide/source/operations_log_format.rst]: ~~excepted that...to ``out``~~ => with the exception of the domain in the common envelope, which is set to ``out``

File: doc/admin-guide/source/operations_log_format.rst [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+   * - Instance ID
+     - `OIO,OPENIO,meta0,1[12159]:`
+   * - Process ID
+     - `12159`
+   * - Thread ID
+     - `1E9A`
+   * - Domain
+     - `access`
+
+The Domain mention it is a line for the access log, let's unpack this.
```
Comment (vdombrovski) [inline doc/admin-guide/source/operations_log_format.rst]: Depending on the carried meaning:
For the access log, the Domain field is a line, that can be unpacked as follows 
**or**
the Domain parameter groups several fields that can unpacked as follows

File: doc/admin-guide/source/operations_log_format.rst [hunk 13]
```diff
@@ -0,0 +1,157 @@
...
+   * - User ID
+     - `-`
+   * - Session ID
+     - `742FBB9DC7674C7C7959957801F06B44`
+   * - Payload
+     - `t=63 AAA0`
+
+In this example, all the fields are always present as expected, but one
+missing fields has been valuated by a dash. The final field is has an arbitrary
+(or unspecified) format, it depends from the service implementation.
```
Comment (vdombrovski) [inline doc/admin-guide/source/operations_log_format.rst]: ~~from~~ => on

**Pull request merged**